docs/source/usage.rst

```diff
@@ -51,6 +51,19 @@ Once the module is imported, you can create a Verto converter creating an Verto
 
 - ``extensions`` -  A list of extra Markdown extensions to run in the converter. Details on how to use this parameter can be found on the :doc:`extensions` page.
 
+- ``custom_argument_rules`` - A json file to modify the default argument rules for each tag. The default rules can found by reading the documentation for each tag.
```

Comment:
This should be a dictionary rather than a JSON file, as I don't want to have to create a file just to be read by Verto. Thoughts?

(It appears to only want a dictionary though from what I'm reading in the code).